### PR TITLE
Bump OVS CNI to 0.29.1 and adapt to OVS CNI based on a single image

### PR DESCRIPTION
**Pull request by phoracek**
**What this PR does / why we need it**:

Since https://github.com/k8snetworkplumbingwg/ovs-cni/pull/244 OVS CNI is shipping all its binaries in a single container image and deploying all CNI plugins via a single init container.

This PR adapts to that change.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
running it locally I see that `ovs-cni-amd-64` is in `Init:CrashLoopBackOff`

File: data/ovs/001-ovs-cni.yaml [hunk 1]
```diff
@@ -30,33 +30,11 @@ spec:
...
-          volumeMounts:
-            - name: cnibin
-              mountPath: /host{{ .CNIBinDir }}
-        - name: ovs-mirror-consumer
-          image: {{ .OvsCNIMirrorConsumerImage }}
-          command: ["cp", "/ovs-mirror-consumer", "/host{{ .CNIBinDir }}/ovs-mirror-consumer"]
+          command: ["/bin/sh", "-c"]
+          args:
+            - >
+              cp /ovs /host/opt/cni/bin/ovs && cp /ovs-mirror-producer /host/opt/cni/bin/ovs-mirror-producer && cp /ovs-mirror-consumer /host/opt/cni/bin/ovs-mirror-consumer
```
[the remark this answers is not in the document]
Comment (RamLavi) [reply data/ovs/001-ovs-cni.yaml:36]: of course you'll need to also add this line to the bumper script:
```
yaml-utils::update_param ${f} spec.template.spec.initContainers[0].args 'cp /ovs /host{{ .CNIBinDir }}/ovs && cp /ovs-mirror-producer /host{{ .CNIBinDir }}/ovs-mirror-producer && cp /ovs-mirror-consumer /host{{ .CNIBinDir }}/ovs-mirror-consumer'
```

[the remark this answers is not in the document]
Comment (phoracek) [reply data/ovs/001-ovs-cni.yaml:36]: Oh yeah, good catch

[the remark this answers is not in the document]
Comment (phoracek) [reply data/ovs/001-ovs-cni.yaml:36]: > running it locally I see that `ovs-cni-amd-64` is in `Init:CrashLoopBackOff`

I think I fixed it in the latest revision, did you use that one?

[the remark this answers is not in the document]
Comment (phoracek) [reply data/ovs/001-ovs-cni.yaml:36]: I tried a different approach - hiding the variable path behind volume. Let me know if that works for you.

[the remark this answers is not in the document]
Comment (RamLavi) [reply data/ovs/001-ovs-cni.yaml:36]: > > running it locally I see that `ovs-cni-amd-64` is in `Init:CrashLoopBackOff`
> 
> I think I fixed it in the latest revision, did you use that one?

yeah it's good now

> I tried a different approach - hiding the variable path behind volume. Let me know if that works for you.

will check

[the remark this answers is not in the document]
Comment (RamLavi) [reply data/ovs/001-ovs-cni.yaml:36]: >I tried a different approach - hiding the variable path behind volume. Let me know if that works for you.

IIUC we already use the mount path, your change simply decoupled the path inside the ovs-cni pod and the actual host path on the node. 
I don't mind doing that. there's no rule it should be consistent. So I'm good

**Pull request merged**